src/lib/parser.rs

```diff
@@ -111,32 +113,67 @@ pub fn parse<TokId: Copy + TryInto<usize>>(grm: &YaccGrammar, stable: &StateTabl
                 debug_assert_eq!(tstack.len(), 1);
                 return Ok(tstack.drain(..).nth(0).unwrap());
             },
-            _ => {
-                return Err(ParseError);
+            None => {
+                let err_la = match la {
+                    Some(x) => *x,
+                    None => {
+                        Lexeme::new(TokId::try_from(usize::from(grm.eof_term_idx()))
+                                          .ok()
+                                          .unwrap(),
+                                    last_la_end, 0)
+                    }
+                };
+                return Err(vec![ParseError{state_idx: st, lexeme: err_la}]);
```

Comment:
This was a "warmup" for returning multiple errors: at this point in the sequence, which lacks recovery, only one error can actually be returned, but the API still wants it to return `Vec<ParseError<TokId>>`.